File: Java/MultiplicationTable.java
// Java program to calculate multiplication table of a given number.

import java.util.Scanner;

class MultiplicationTable{
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        
        System.out.print("Enter the number : ");
        String x = sc.next();
        int n = Integer.parseInt(x);

        System.out.println("The multiplication table of "+n+" is");
        for(int i=1; i<=10; i++){
            System.out.println(n+" * "+i+" = "+n*i);
        }
    }
}

File: Python/multiplication_table.py
# Python program to print multiplication table of the entered number. 

n = int(input('Enter the number of which you want to print the table : '))

print('Multiplication table of '+str(n)+ ' : ')
for i in range(1, 11):
    print(str(n)+' * '+str(i)+' = '+str(n*i))
    # print(f'{n} * {i} = {n*i}')

# Another way of printing the result using f string or string literals
# print(f'{n} * {i} = {n*i}')
